### ISSUE TITLE
Completion doesn't suggest members with no modifier

### ISSUE THREAD

**Issue by nitishch** [jdee-emacs/jdee#93]
Say two classes A, B are in the same package. When I'm in file B.java, I should be able to access members from A that have no access modifier specified.

In such case, completion only suggests public members of A.

**Comment by pwojnowski** [comment]
Are the classes compiled?

**Comment by nitishch** [comment]
Yes. Does it work for the described scenario in your system?

**Comment by pwojnowski** [comment]
IIUC the scenario is as follows:```
// Another.java:
package poligon;
public class Another {
    int sum(int a, int b) {
        return a + b;
    }
}
// Poligon.java:
package poligon;
public class Poligon {
    public static void main(String[] args) {
        new Poligon().action();
    }
    void action() {
        Another a = new Another();
        System.out.println(a.sum(2, 3));
    }
}
```
In "action" method, "sum" is shown in the completion list for "a.". Is this the same case?

**Comment by nitishch** [comment]
Yes. For some reason, this doesn't work in my system. I don't have access to it now. Once I do, I'll see more about this issue.
